cli/packages/gateway/connection.go

```
@@ -0,0 +1,107 @@
+		defer wg.Done()
+		_, err := io.Copy(dst, src)
+		if err != nil && !errors.Is(err, io.EOF) {
+			log.Error().Msgf("Copy error: %v", err)
+		}
+
+		// Signal we're done writing
+		done <- true
+
+		// Half close the connection if possible
+		if c, ok := dst.(CloseWrite); ok {
+			c.CloseWrite()
```

golangci-lint (1.62.2)

Error return value of `c.CloseWrite` is not checked

(errcheck)